_includes/pbc/all/install-features/202307.0/install-dynamic-multistore.md

```diff
@@ -1097,6 +1166,7 @@ class StoreDependencyProvider extends SprykerStoreDependencyProvider
             new DefaultLocaleStorePostCreatePlugin(),
             new LocaleStorePostCreatePlugin(),
             new SearchSetupSourcesStorePostCreatePlugin(),
+            new ContextStorePostCreatePlugin(),
```

Comment:
Verification step is missing (for mostly all new plugins in the PR), each plugin must have a verification step, please fix.